lib/cassandra_migrations/cql-rb-wrapper.rb

```diff
@@ -34,13 +34,12 @@ def add(*args)
 
 class Client
   def self.connect(options)
-    if defined?(Spring)
-      Spring.after_fork do
-        @cluster = Cassandra.cluster(options)
-      end
-    else
-      @cluster = Cassandra.cluster(options)
+    @cluster = Cassandra.cluster(options)
```

Comment:
To make sure: here we remove the Spring after_fork logic, as it wasn't working (@cluster was always nil, I assume since all this call does is register the block for the callback), and if I read the ruby-driver docs correctly, this after_fork should be done on the session, not the cluster.

http://datastax.github.io/ruby-driver/features/#i-get-deadlock-detected-errors